integration/generic-tests/ip_test.go

```diff
@@ -166,6 +162,29 @@ func TestIP(t *testing.T) {
         ip tunnel del ipip_tunnel || exit 1
         ! grep -q "ipip_tunnel" /proc/net/dev || exit 1
 
+        # Add a neighbor (ARP entry) on eth0
+		cat /proc/net/arp
+		ip neigh add 192.168.1.2 lladdr 00:11:22:33:44:55 dev eth0 || exit 1
+		grep -q "192.168.1.2" /proc/net/arp || exit 1
+
+		# Verify the neighbor entry
+		ip neigh show dev eth0 || exit 1
+		neigh_entry=$(ip neigh show dev eth0 | grep "192.168.1.2")
+		test "$neigh_entry" = "192.168.1.2 dev eth0 lladdr 00:11:22:33:44:55 PERMANENT" || exit 1
+
+		# Replace the entry with different flags
```

Comment:
It might be me, but 'different flags' sounds quite opaque to me; it is not clear what this means. Can you add a short comment about how it differs?

Reply:
Added a more clear description of all the flags.